core/control-plane/control-plane-api-client/src/main/java/org/eclipse/edc/connector/api/client/transferprocess/TransferProcessHttpClient.java

```diff
@@ -11,7 +11,6 @@
  *       Microsoft Corporation - initial API and implementation
  *
  */
-
```

Comment:
Please remove this edit as it pollutes this diff. Thx

Reply:
Hello @jimmarino, no problem.